src/surfaces/filtersRail/filtersRail.jsx

```diff
@@ -181,7 +186,11 @@ function FiltersRail(props) {
                         },
                     )}
                 >
-                    {children}
+                    <ScrollBar autoHide>
```

Comment:
A thought: could we setup a prop (`isScrollable`) and have logic for implementing this scrollbar?

Reply:
Yep, I'll make it so.

Reply:
Made it so in ed6c63a.